src/sass/list/list.scss

```diff
@@ -0,0 +1,96 @@
+@import "../variables/variables";
+@import "../mixins/private/button-mixins";
+@import "../mixins/private/token-mixins";
+
+.list {
+    max-width: 480px;
+}
+
+.list ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.list ul li > * {
+    @include background-color-token(
+        list-background-color,
+        color-background-primary
+    );
+
+    align-items: center;
+    box-sizing: border-box;
+    color: var(--color-foreground-on-primary);
+    display: inline-flex;
+    font-size: var(--font-size-16);
+    margin: 1px;
+    min-height: var(--spacing-600);
+    padding: var(--spacing-150) var(--spacing-200);
+    width: 100%;
+}
+
+.list__leading {
+    margin-right: var(--spacing-200);
```

Comment:
For all of these `margins`, we should probably use `margin-inline-x` and `margin-block-x`.